tools/name.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# Make sure you have python3 installed.
+# Ensure that the json_formatter is kept in Tools with this script.
+# They must be in the same folder!
+# For Windows:
+# Using command prompt type "python barrellength_volume.py"
+# For Max OS X or Linux:
+# Swap any "\\" with "/", then run the script as in windows.
+
+import json
+import os
+
+
+def gen_new(path):
+    change = False
+    with open(path, "r") as json_file:
+        try:
+            json_data = json.load(json_file)
+        except UnicodeDecodeError:
+            print("UnicodeDecodeError in {0}".format(path))
+            return None
+        except json.decoder.JSONDecodeError:
+            print("JSONDecodeError in {0}".format(path))
+            return None
+        for jo in json_data:
+            # We only want JsonObjects
+            if type(jo) is str:
+                continue
+            if not jo.get('name'):
+                continue
+            if type(jo['name']) == dict:
+                continue
+            if jo.get['type'] not in ['AMMO', 'ARMOR', 'BATTERY', 'bionic',
```

Comment:
Did you mean `jo.get('type')`? What's this list for? I suggest moving it into a global variable.

Reply:
I tried this it doesn't work

Reply:
it does work